docs/nebula-design-concepts.md

```diff
@@ -0,0 +1,216 @@
+# Data modeling and design concept
+------
+**Nebula Graph** is comprised of four components: storage layer, metadata service, compute layer and client console. In this documentation, we'll discuss data modeling and architecture of Nebula Graph.
+
+---
+## Directed property graph
+
+Modeling data with directed property graph, Nebula Graph is very easy to understand.
+That is to say, Nebula Graph consists of two graph elements: vertex and edge. Each
+vertex represents an entity, and each edge represents how two vertices are associated.
+
+![image][example1]
+
+<!-- ### Vertex
+
+Vertices are typically used to store entity information. In the preceding example, vertices are **players** and **team**.
+
+### Edges
+
+Edges are used to connect vertices to one another and therefore provide a means
```

Comment:
errrr. Can u rewrite this sentence?

Reply:
I'm sorry I don't see any problems here.

Reply:
done

Reply:
> Edges are used to connect vertices. They usually represent a relationship (i.e. ownership, friendship, and so on) or a behavior (i.e. transfer, like, and so on) between vertices. In **Nebula Graph**, an edge is always directional and of a specific edge type. But between any two vertices, there could be multiple edges of the same edge type. These edges will be differentiated by their rankings. So, any edge is uniquely identified by the tuple `[src_vertex, dst_vertex, edge_type, ranking]`

I like the way you are saying, have mixed yours with mine.

Reply:
What about  _But between any two vertices_ => _However, from the source vertex to the target vertex_   _So_ => _That's to say_